modules/setting/setting.go

```diff
@@ -711,6 +716,12 @@ func NewContext() {
 		} else if err = os.MkdirAll(SSH.KeyTestPath, 0644); err != nil {
 			log.Fatal("Failed to create '%s': %v", SSH.KeyTestPath, err)
 		}
+
+		trustedUserCaKeys := sec.Key("SSH_TRUSTED_USER_CA_KEYS").Strings(",")
+		if err := ioutil.WriteFile(filepath.Join(SSH.RootPath, "trusted-user-ca-keys.pem"),
+			[]byte(strings.Join(trustedUserCaKeys, "\n")), 0600); err != nil {
+			log.Fatal("Failed to create '%s': %v", filepath.Join(SSH.RootPath, "trusted-user-ca-keys.pem"), err)
+		}
```

Comment:
Hmm... This might blast away important settings. I'm not sure what to do here. 

Clearly we should only do this if we're going to create the authorized principals file, but we should also only do this if we are sure that we have all of the CA keys that we want to trust.

Reply:
I don't understand the problem? What do you mean it might blast away important settings?